content/cloud-docs/run/states.mdx

```diff
@@ -16,11 +16,13 @@ In the list of workspaces on Terraform Cloud's main page, each workspace shows t
 _States in this stage:_
 
 - **Pending:** Terraform Cloud hasn't started action on a run yet. Terraform Cloud processes each workspace's runs in the order they were queued, and a run remains pending until every run before it has completed.
```

Comment:
Do we need to say something at the beginning of this section about how like VCS automatically discards your file? Kind of like what you have here in the API docs:

"Terraform Cloud automatically archives configuration versions created through VCS when associated runs are complete and then re-fetches the files for subsequent runs."

I just think that might help users understand the new states we added like "fetching"

Reply:
Oh hmm good call - I was trying to keep it succinct and not over-explain but I might have gone too far in the other direction 😆

Reply:
How's [this](https://github.com/hashicorp/terraform-website/pull/2157/commits/fe9977aa7e47e5e11e55d1b2506c61f7f8101ea9#diff-57435ad5015080604531b6d160ba221a0c65fd05bc606fc75932b9a751cc8070R16)?

Reply:
YESSSS I like that a lot :-)